README.md

```diff
@@ -234,50 +189,29 @@ upload the generated `extension-src.zip`:
 
 ### Demoing the platform
 
-#### OSX (mac)
-
 This supposes that you have followed the instruction from the rest of this README.
 
 ##### First time
 
 Suggested steps:
-1. Open 4 tabs in a terminal and `cd` into this repository for each tab
-2. In the first tab run `npm run redis`.
-3. In the second tab run `npm run cloudant`
-4. In the third tab:
+1. Start the local services using `docker-compose`.
+1. In a shell attached to the 'web' container:
    - run `npm run seed` or `npm run reset` to either seed the database with
      sample data (or start from a clean state)
    - run `npm start` to start the web server
 5. In the fourth tab run `npm run extension:watch` and update the extension in
    your browser (see section above for instructions)
 6. You can now visit [http://127.0.0.1:3000/](http://127.0.0.1:3000/) and give a demo
 
-When you are done with the demo do:
-
-1. in the fourth tab (extension watcher) run `ctrl + c` to kill the node process.
-2. in the third tab (web server) run `ctrl + c` to kill the node process. To be
-   sure you can also run `killall node` to be sure that no zombie node processes
-   remain.
-3. in the second tab (cloudant) nothing to do (you can quit docker if you are done using it)
-4. in the first tab (redis) run `ctrl + c`
-
-Everything should now be shut down.
-
-##### Subsequent times
-
-1. Open 3 tabs in a terminal and `cd` into this repository for each tab
-2. In the first tab run `npm run redis`
-3. In the second tab run `npm run cloudant`
-4. In the third tab run `npm start`
+When you are done with the demo you can use `docker-compose down` to shut down the server.
 
 ##### Updating your local install
 
 1. `cd` into this repository
 2. run `git fetch` followed by `git merge origin/master`
-3. run `npm install`
+3. Connect a shell to the web container and run `npm install`
```

Comment:
Can't this be done on host?

Reply:
Yes, however, I think the readme should recommend using node in the container to avoid issues arising from differences between the host machine and the node environment that actually runs the service.  In particular, I don't think we should require Node to be installed at all on the host machine.